eth/tools/fixtures/helpers.py

```diff
@@ -156,7 +161,7 @@ def chain_vm_configuration(fixture: Dict[str, Any]) -> Iterable[Tuple[int, Type[
             (0, SpuriousDragonVM),
             (5, ByzantiumVM),
         )
-    elif network == 'ByzantiumToConstantinopleAt5':
+    elif network == 'ByzantiumToConstantinopleFixAt5':
```

Comment:
This looks suspicious, the previous name `ByzantiumToConstantinopleAt5` was correctly reflected with the selected VMs `ByzantiumVM` and `ConstantinopleVM`. But now the transition seems to go from `Byzantium` to `ConstantinopleFix` where `ConstantinopleFix` is the `ethereum/test` way to refer to `PetersburgVM`.

Reply:
Thanks for catching this. I need to verify this again. But this was something that I had changed just to get the tests passing. Since the tests didn't complain, I didn't forget to verify this change. If the tests have in fact been renamed then I need to change line 167 to `PetersburgVM ` 👍 . I will verify this and make sure that I haven't missed any case(since tests didn't raise ValueError, I assume I didn't).

Reply:
I've grepped `fixtures`, and there are no longer instances of string `ByzantiumToConstantinopleAt5` there.

The upstream tests check for main-net transition from `ByzantiumVM` to `PetersburgVM`, it seems. These two:

```
fixtures/BlockchainTests/TransitionTests/bcByzantiumToConstantinopleFix/ConstantinopleFixTransition.json:        "network" : "ByzantiumToConstantinopleFixAt5",
fixtures/src/BlockchainTestsFiller/TransitionTests/bcByzantiumToConstantinopleFix/ConstantinopleFixTransitionFiller.json:	    "network" : "ByzantiumToConstantinopleFixAt5",
```